src/plugin/Manager.cc

```diff
@@ -223,17 +227,17 @@ bool Manager::ActivateDynamicPluginInternal(const std::string& name, bool ok_if_
 			if ( util::strtolower(current_plugin->Name()) != util::strtolower(name) ) {
 				errors->push_back(util::fmt("inconsistent plugin name: %s vs %s",
 						     current_plugin->Name().c_str(), name.c_str()));
-				return false;
+				continue;
 			}
 
-			current_dir = nullptr;
-			current_sopath = nullptr;
 			current_plugin = nullptr;
-
 			DBG_LOG(DBG_PLUGINS, "  Loaded %s", path);
 			}
 
 		globfree(&gl);
+
+		if ( errors->size() )
```

Comment:
```suggestion
		if ( ! errors->empty() )
```

Reply:
I seem to be the only person who prefers `.size()` over `! empty()`. :-) Will change.